routes/web-admin.php

```diff
@@ -223,3 +223,14 @@
         'uses' => 'VisitorStatsController@index',
     ]);
 });
+
+Route::group([
```

Comment:
Can you format this using the `->uses()`  and `->name()` helpers please? This will allow easier merging with #1219.